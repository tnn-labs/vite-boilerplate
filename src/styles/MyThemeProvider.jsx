import { ThemeProvider } from 'styled-components';
import { theme } from './theme';
import PropTypes from 'prop-types';
import { GlobalStyles } from './global-styles';

export const MyThemeProvider = ({ children }) => {
  return (
    <ThemeProvider theme={theme}>
      {children}
      <GlobalStyles />
    </ThemeProvider>
  );
};

MyThemeProvider.propTypes = {
  children: PropTypes.node.isRequired,
};
