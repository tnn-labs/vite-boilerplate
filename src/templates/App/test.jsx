import { screen } from '@testing-library/react';
import { App } from '.';
import { renderTheme } from '../../styles/render-theme';

it('should test', () => {
  renderTheme(<App />);
  expect(screen.getAllByRole('heading')).toMatchSnapshot();
});
