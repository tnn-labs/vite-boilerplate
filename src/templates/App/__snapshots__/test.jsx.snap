// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test 1`] = `
Array [
  .c0 {
  background: red;
}

<h1
    class="c0"
  />,
]
`;
